assets/src/blocks/SocialMedia/SocialMediaEmbed.js

```diff
@@ -1,42 +1,43 @@
+/* eslint-disable jsx-a11y/iframe-has-title */
```

Comment:
Here again I think that we could add simple titles to the iframes instead of using this

Reply:
Got it.